eng/scripts/Publish-SwiftMirrorRedirectReadme.ps1

```diff
@@ -0,0 +1,56 @@
+param (
+    [Parameter(Mandatory = $true)]
+    [ValidateNotNullOrEmpty()]
+    [string] $GitSourcePath,
```

Comment:
Looks like you only use the readme file, so might be worth just passing in that path instead of the entire repo path.

Reply:
I'll do a follow-on PR with these updates.